### ISSUE TITLE
Compilation warning

### ISSUE THREAD

**Issue by VictorLamoine** [grynko/curlite#1]
Ubuntu 14.04
gcc 4.9.4

``` cpp
curlite.hpp:298:57: warning: comparison between ‘enum curlite::OptionLongCode::<anonymous>’ and ‘enum curlite::OptionInvalidCode::<anonymous>’ [-Wenum-compare]
         static_assert( OptionTypeCode<ValueType>::value != OptionInvalidCode::value, "the type is not supported by curl_easy_setopt" );

```

**Comment by grynko** [comment]
Hi Victor,

I'm sorry for this long delay in my response. The issue is fixed after d91a907.

Thank you.

**Comment by VictorLamoine** [comment]
It does fix the warning thank you!
